srcs/execute/env_free.c

```diff
@@ -0,0 +1,51 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        ::::::::            */
+/*   env_free.c                                         :+:    :+:            */
+/*                                                     +:+                    */
+/*   By: jsaariko <jsaariko@student.codam.nl>         +#+                     */
+/*                                                   +#+                      */
+/*   Created: 2020/10/29 10:59:48 by jsaariko      #+#    #+#                 */
+/*   Updated: 2020/10/29 14:01:27 by jsaariko      ########   odam.nl         */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "execute.h"
+#include "error.h"
+#include "libft.h"
+
+void			free_env_item(t_env *env)
+{
+	free(env->key);
+	free(env->value);
+	free(env);
+}
+
+void			free_environment(t_vector *env)
+{
+	size_t	i;
```

Comment:
The free functions have been made but not called anywhere yet. Do they actually need to be? Or is the environment something that persists as long as the shell is running?